### ISSUE TITLE
Client Only Components Example has a Dependency Issue

### ISSUE THREAD

**Issue by ntorba** [remix-run/examples#150]
When running `npm i` for the [client-only components example](https://github.com/remix-run/examples/tree/main/client-only-components), I get the following output:

```
(base) nick@nick-Laptop:~/Documents/GitHub/examples/client-only-components$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: undefined@undefined
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR!   peer react@">=16.8" from @remix-run/react@1.12.0
npm ERR!   node_modules/@remix-run/react
npm ERR!     @remix-run/react@"*" from the root project
npm ERR!     peer @remix-run/react@"^1.1.1" from remix-utils@2.11.1
npm ERR!     node_modules/remix-utils
npm ERR!       remix-utils@"^2.7.0" from the root project
npm ERR!   1 more (react-dom)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from remix-utils@2.11.1
npm ERR! node_modules/remix-utils
npm ERR!   remix-utils@"^2.7.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/nick/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/nick/.npm/_logs/2023-02-07T17_04_21_010Z-debug-0.log
```

Can someone help me update this?

**Comment by dwfee** [comment]
just install remix-utils it works then!

**Comment by MichaelDeBoey** [comment]
This is fixed by #161
